src/openzaak/management/commands/dump_auth_fixtures.py

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: EUPL-1.2
```

Comment:
You know it looks like you just reuse the existing command without modifying it. I think it could better be a  bash file, just like `bin/generate_admin_index_fixture.sh`, which we already have.

Reply:
Hmm yeah I'll move it to a bash script, though I do think at some point we'll want to be able to export from the admin interface, but we can just do `call_command` with `dumpdata` in that case